build/common.sh

```diff
@@ -219,16 +219,21 @@ function kube::build::docker_available_on_osx() {
 
 function kube::build::prepare_docker_machine() {
   kube::log::status "docker-machine was found."
+
+  local available_memory=$(sysctl -n hw.memsize 2>/dev/null)
+  local virtualbox_memory=$(( (${available_memory} * 1024) / (1073741824 * 2) ))
```

Comment:
why are you dividing available memory by `1073741824 * 2`?

Reply:
Great question! `sysctl -n hw.memsize` outputs bytes and virtualbox expects mb for the `virtualbox-memory` flag. Looking at it again, its probably not the most efficient calculation, but I wanted to assign virtualbox half the available system memory. I will consider how to express this more succintly (and improve the variable names accordingly).

Do you think giving virtualbox have the system memory is appropriate, or would you give it all?

Reply:
ah, that make sense. with a comment explaining what you wrote here, I think this is fine.

Reply:
Perfect, will fix both tomorrow morning. Thanks @ixdy :)